.github/workflows/pr-labeler.yml

```diff
@@ -0,0 +1,51 @@
+name: PR Labeler
+on: [pull_request]
+
+jobs:
+  label:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/github-script@v6
+      with:
+        github-token: ${{secrets.GITHUB_TOKEN}}
+        script: |
+          const fs = require('fs');
+          const path = require('path');
+
+          function getFileExtensions(files) {
+            return files.map(file => path.extname(file).toLowerCase()).filter(ext => ext !== '');
+          }
+
+          async function addLabels(extensions) {
+            const labels = [];
+            const CONTENT_EXTENSIONS = ['.mdx', '.md', '.svg', '.jpg', '.png'];
+            const CODE_EXTENSIONS = ['.tsx,', '.ts', '.html', '.css', '.json', '.toml', '.lock', '.yml', '.yaml', '.js'];
```

Comment:
I don't think this is needed. We should only have to scan for any .mdx changes and add the content label based on that. Any other changes default to code.

Reply:
Yeah, ok, keep the `.md` !